src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+  let key = '';
+  let value = '';
+
+  const splitAllstring = sourceString.split(';');
+
+  for (const block of splitAllstring) {
+    const splitBlock = block.split(':');
+
+    key = splitBlock[0];
+    value = splitBlock[1];
+
+    if (value === undefined) {
+      continue;
+    }
+    resultObject[key.trim()] = value.trim();
```

Comment:
Instead of using a conditional 'if' statement and 'continue', you can filter out any undefined values before the loop begins. This can be done using the 'filter' method on the array of style declarations.